lib/ProductCard/ProductCard.js

```diff
@@ -0,0 +1,96 @@
+import clsx from 'clsx';
+import { ImageGallery } from '../ImageGallery';
+import { StarRating } from '../StarRating';
+
+const ProductCard = (props) => {
+  const {
+    classNames = {},
+    title,
+    price,
+    url,
+    imageUrl,
+    altImage,
+    Image: ImageComponent,
+    starRating = 0,
+    reviews,
+    onAddToCart,
+    displayVariants = false,
+    variantImages = [],
+  } = props;
```

Comment:
This is peak OCD (sorry) but let's order these prop names alphabetically. Same for `ImageGallery`. 🙏🏻